Library/Formula/nedit.rb

```diff
@@ -0,0 +1,39 @@
+class Nedit < Formula
+  homepage "http://www.nedit.org"
+  url "https://downloads.sourceforge.net/project/nedit/nedit-source/nedit-5.6-src.tar.gz"
+  sha1 "133641224b0969e31e3404b874c71c35341f028b"
+
+  depends_on :x11
+  depends_on "openmotif"
```

Comment:
I couldn’t find any formula for openmotif in the main Homebrew repo, did I miss something?